CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ All notable changes to this project will be documented in this file.
 * Fix:
 	* Retain cycle in example project
 
+## [1.3.0]
+* Feature:
+        * Exposed new delegate method `playerContext(errorCode: Int, errorMessage: String?)`
```

Comment:
say what's the purpose of the feature, instead
example: "Get item error logs on playing"

Reply:
Done